proto/decentraland/social/friendships/friendships.proto

```diff
@@ -6,14 +6,14 @@
 syntax = "proto3";
 package decentraland.social.friendships;
 
-// Validation and anti-spam errors
-enum FriendshipErrorCode {
-  TOO_MANY_REQUESTS_SENT = 0;
-  NOT_ENOUGH_TIME_PASSED = 1;
-  BLOCKED_USER = 2;
-  NON_EXISTING_USER = 3;
-  INVALID_REQUEST = 4;
-  UNKNOWN = 5;
+enum ServiceErrors {
+  UNKNOWN = 0;
```

Comment:
should unknown be the same as internal server error?

Reply:
Yes, u r right :)